README.md

```diff
@@ -74,7 +74,7 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v2
     - name: IIS stop
-      uses: 'im-open/iis-service-action@v2.0.0'
```

Comment:
A downgrade? from 2.0.0 to 1.0.5?

Reply:
I'm not sure what happened with this repository, but there isn't currently a `v2.0.0`.